partials/_name.html.erb

```diff
@@ -1,5 +1,3 @@
 <h1>hello <%= name %></h1>
 
-<% 50.times do %>
-  <%= render '/nested_name', name: name %>
-<% end %>
+<%= render partial: '/nested_name', collection: [name] * 50, as: :name %>
```

Comment:
does using `cached: true` here make sense? or would it not make any difference? or possibly slow it down?

Reply:
When I run it with `cached: true` it slows down.

Reply:
i imagine it computing the cache key is whats slowing it down, and since theres no DB or anything, nothing worth caching. thanks!

Reply:
Wait, are we allowed to use caching?

Reply:
@joeldrapper i have no idea. The rules are made up...I'm assuming Rails production env is going to use some level of view caching?? Particularly for template lookups and maybe previously compiled templates?